catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/XmlInputTransformer.java

```diff
@@ -288,10 +293,10 @@ public String getName() {
 
     @Override
     public AttributeDescriptor getAttributeDescriptor(String attributeName) {
-        return attributeDescriptors.stream()
```

Comment:
This was a change done by @harrison-tarr for DDF-899, which I carried over. Purpose is to prevent an NPE from being thrown in the case that the attribute doesn't have a descriptor.